Userland/Libraries/LibJS/Tests/builtins/Temporal/PlainDateTime/PlainDateTime.prototype.add.js

```diff
@@ -5,8 +5,8 @@ describe("correct behavior", () => {
 
     test("basic functionality", () => {
         const plainDateTime = new Temporal.PlainDateTime(1970, 1, 1);
-        const result = plainDateTime.add(new Temporal.Duration(51, 6, 5, 18, 14, 47));
-        expect(result.equals(new Temporal.PlainDateTime(2021, 7, 6, 18, 14, 47)));
+        const result = plainDateTime.add(new Temporal.Duration(51, 6, 0, 5, 18, 14, 47));
```

Comment:
Should probably be in the third commit (bad rebase?)

Reply:
oops, fixed!